<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="19" passed="3" failed="16" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-28T19:01:55 IST" name="Default suite" finished-at="2024-12-28T20:57:56 IST" duration-ms="6961592">
    <groups>
    </groups>
    <test started-at="2024-12-28T19:01:55 IST" name="Default test" finished-at="2024-12-28T20:57:56 IST" duration-ms="6961592">
      <class name="com.test.user.UserManagementModule.UserManagement_TC">
        <test-method is-config="true" signature="reference()[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:00:12 IST" name="reference" finished-at="2024-12-28T19:01:55 IST" duration-ms="102353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reference -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:01:55 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:01:55 IST" duration-ms="255" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Navigation_to_User_Management_Page() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Navigation_to_User_Management_Page()[pri:1, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:01:55 IST" name="Navigation_to_User_Management_Page" finished-at="2024-12-28T19:02:16 IST" duration-ms="20896" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Navigation_to_User_Management_Page -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:16 IST" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Navigation_to_User_Management_Page status=SUCCESS method=UserManagement_TC.Navigation_to_User_Management_Page()[pri:1, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method signature="User_search_page__Default_State_of_IsActive_Checkbox()[pri:2, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="User_search_page__Default_State_of_IsActive_Checkbox" finished-at="2024-12-28T19:02:16 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null
at com.Pages_UserManagementModule.UserManagement_Page.isIsActiveCheckboxSelectedOrNot(UserManagement_Page.java:109)
at com.test.user.UserManagementModule.UserManagement_TC.User_search_page__Default_State_of_IsActive_Checkbox(UserManagement_TC.java:108)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- User_search_page__Default_State_of_IsActive_Checkbox -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:16 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.User_search_page__Default_State_of_IsActive_Checkbox() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:16 IST" duration-ms="180" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=User_search_page__Default_State_of_IsActive_Checkbox status=FAILURE method=UserManagement_TC.User_search_page__Default_State_of_IsActive_Checkbox()[pri:2, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:16 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.User_search_page__Pagination_of_Active_users() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="User_search_page__Pagination_of_Active_users()[pri:3, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:16 IST" name="User_search_page__Pagination_of_Active_users" finished-at="2024-12-28T19:02:17 IST" duration-ms="622" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- User_search_page__Pagination_of_Active_users -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:17 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=User_search_page__Pagination_of_Active_users status=SUCCESS method=UserManagement_TC.User_search_page__Pagination_of_Active_users()[pri:3, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:17 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking()[pri:4, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking" finished-at="2024-12-28T19:02:17 IST" duration-ms="241" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.ClickUserManagementSecondPageBtn(UserManagement_Page.java:161)
at com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking(UserManagement_TC.java:162)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:17 IST" duration-ms="177" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking status=FAILURE method=UserManagement_TC.User_search_page_Pagination_of_Active_users_Second_page_button_functionality_and_data_checking()[pri:4, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method signature="User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking()[pri:5, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking" finished-at="2024-12-28T19:02:23 IST" duration-ms="5677" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.ClickUserManagementLastArrowBtn(UserManagement_Page.java:152)
at com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking(UserManagement_TC.java:179)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:17 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:17 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:23 IST" duration-ms="184" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking status=FAILURE method=UserManagement_TC.User_search_page_Pagination_of_Active_users_Last_page_button_functionality_and_data_checking()[pri:5, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:23 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking()[pri:6, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking" finished-at="2024-12-28T19:02:23 IST" duration-ms="311" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.ClickUserManagementPreviousArrowBtn(UserManagement_Page.java:170)
at com.test.user.UserManagementModule.UserManagement_TC.User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking(UserManagement_TC.java:194)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:23 IST" duration-ms="157" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking status=FAILURE method=UserManagement_TC.User_search_page_Pagination_of_Active_users_Previous_page_button_functionality_and_data_checking()[pri:6, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:23 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Test_Redirect_to_Add_New_User_Page() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Test_Redirect_to_Add_New_User_Page()[pri:7, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:23 IST" name="Test_Redirect_to_Add_New_User_Page" finished-at="2024-12-28T19:02:24 IST" duration-ms="336" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.ClickUserManagementPageAddUserBtn(UserManagement_Page.java:179)
at com.test.user.UserManagementModule.UserManagement_TC.Test_Redirect_to_Add_New_User_Page(UserManagement_TC.java:208)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Redirect_to_Add_New_User_Page -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:24 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:24 IST" duration-ms="196" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Test_Redirect_to_Add_New_User_Page status=FAILURE method=UserManagement_TC.Test_Redirect_to_Add_New_User_Page()[pri:7, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:24 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:24 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible()[pri:8, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:24 IST" name="Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible" finished-at="2024-12-28T19:02:24 IST" duration-ms="505" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:24 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:24 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible status=SUCCESS method=UserManagement_TC.Page_redirects_to_Add_New_User_page_showing_Name_Email_Phone_number_fields_Role_and_Organization_Type_dropdowns_Close_and_Submit_buttons_are_visible()[pri:8, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:26 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:26 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown(java.util.Map)[pri:9, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:26 IST" name="Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown" data-provider="TestData" finished-at="2024-12-28T19:02:39 IST" duration-ms="13449" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.SelectOrganizationTypeDropdown(UserManagement_Page.java:234)
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown(UserManagement_TC.java:274)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:39 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:39 IST" duration-ms="165" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown status=FAILURE method=UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Head_office_Display_Head_Office_Dropdown(java.util.Map)[pri:9, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:39 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:39 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields(java.util.Map)[pri:10, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:39 IST" name="Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields" data-provider="TestData" finished-at="2024-12-28T19:02:43 IST" duration-ms="3745" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.SelectOrganizationTypeDropdown2(UserManagement_Page.java:243)
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields(UserManagement_TC.java:295)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:43 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:43 IST" duration-ms="203" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields status=FAILURE method=UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Zone_Co_Display_Head_office_and_Zone_CO_dropdown_fields(java.util.Map)[pri:10, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:44 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:44 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields(java.util.Map)[pri:11, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:44 IST" name="Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields" data-provider="TestData" finished-at="2024-12-28T19:02:47 IST" duration-ms="3788" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.SelectOrganizationTypeDropdown3(UserManagement_Page.java:250)
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields(UserManagement_TC.java:321)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:47 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:48 IST" duration-ms="232" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields status=FAILURE method=UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Region_Display_Head_office_and_Zone_CO_Region_dropdown_fields(java.util.Map)[pri:11, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:48 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:48 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields(java.util.Map)[pri:12, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:48 IST" name="Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields" data-provider="TestData" finished-at="2024-12-28T19:02:51 IST" duration-ms="3737" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at com.BasePackage.Common.waitForSpinnerToDisappear(Common.java:44)
at com.Pages_UserManagementModule.UserManagement_Page.SelectOrganizationTypeDropdown4(UserManagement_Page.java:257)
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields(UserManagement_TC.java:351)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:51 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:02:52 IST" duration-ms="189" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields status=FAILURE method=UserManagement_TC.Add_new_user_page_Organization_Type_Selection_as_Branch_Display_Head_office_and_Zone_CO_Region_and_Branch_dropdown_fields(java.util.Map)[pri:12, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:52 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:02:52 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Submit_Without_Inputs(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Submit_Without_Inputs(java.util.Map)[pri:13, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:02:52 IST" name="Add_new_user_page_Submit_Without_Inputs" data-provider="TestData" finished-at="2024-12-28T19:12:52 IST" duration-ms="600600" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$13]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$13
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Submit_Without_Inputs(UserManagement_TC.java:394)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$13
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Submit_Without_Inputs -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:12:52 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:12:52 IST" duration-ms="151" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Submit_Without_Inputs status=FAILURE method=UserManagement_TC.Add_new_user_page_Submit_Without_Inputs(java.util.Map)[pri:13, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:12:52 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:12:52 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button(java.util.Map)[pri:14, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:12:52 IST" name="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button" data-provider="TestData" finished-at="2024-12-28T19:25:53 IST" duration-ms="780150" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$14]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$14
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button(UserManagement_TC.java:428)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$14
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:25:53 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:25:53 IST" duration-ms="248" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button status=FAILURE method=UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Name_Field_and_click_on_submit_button(java.util.Map)[pri:14, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:25:53 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:25:53 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button(java.util.Map)[pri:15, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:25:53 IST" name="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button" data-provider="TestData" finished-at="2024-12-28T19:48:53 IST" duration-ms="1380447" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$15]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$15
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button(UserManagement_TC.java:463)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$15
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:48:53 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T19:48:54 IST" duration-ms="339" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button status=FAILURE method=UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Email_Field_and_click_on_submit_button(java.util.Map)[pri:15, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:48:54 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T19:48:54 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button(java.util.Map)[pri:16, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T19:48:54 IST" name="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button" data-provider="TestData" finished-at="2024-12-28T20:11:54 IST" duration-ms="1380330" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$16]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$16
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button(UserManagement_TC.java:497)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$16
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:11:54 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T20:11:55 IST" duration-ms="374" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button status=FAILURE method=UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_Phone_number_Field_and_click_on_submit_button(java.util.Map)[pri:16, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:11:55 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T20:11:55 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button(java.util.Map)[pri:17, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:11:55 IST" name="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button" data-provider="TestData" finished-at="2024-12-28T20:34:55 IST" duration-ms="1380062" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$17]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$17
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button(UserManagement_TC.java:532)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$17
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:34:55 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T20:34:55 IST" duration-ms="360" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button status=FAILURE method=UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_role_dropdown_and_click_on_submit_button(java.util.Map)[pri:17, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:34:55 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T20:34:55 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button(java.util.Map)[pri:18, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:34:55 IST" name="Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button" data-provider="TestData" finished-at="2024-12-28T20:57:55 IST" duration-ms="1380328" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[com/test/user/UserManagementModule/UserManagement_TC$18]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: com/test/user/UserManagementModule/UserManagement_TC$18
at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button(UserManagement_TC.java:563)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: com.test.user.UserManagementModule.UserManagement_TC$18
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:57:55 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T20:57:56 IST" duration-ms="355" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button status=FAILURE method=UserManagement_TC.Add_new_user_page_Enter_and_select_inputs_for_all_fields_except_organization_type_dropdown_and_click_on_submit_button(java.util.Map)[pri:18, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:57:56 IST" name="ExtentTestManagerStartTest" finished-at="2024-12-28T20:57:56 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Invalid_Email_Format(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Add_new_user_page_Invalid_Email_Format(java.util.Map)[pri:19, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:57:56 IST" name="Add_new_user_page_Invalid_Email_Format" data-provider="TestData" finished-at="2024-12-28T20:57:56 IST" duration-ms="2" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=Mahalinga, AlphaNumericInputName=John1234, Branch=Indore I, Email=mahalinga@gmail.com, EmailWithoutDomain=kishan@, HeadOffice=Mumbai, InvalidEmailId=kishangmail.com, InvalidUserName=InvalidUser, Name=Simran, NumberEightDigits=90089257, NumberWithLetters=AbCd, NumericInputName=12345, OrganizationType=Head office, OrganizationType2=Zone/CO, OrganizationType3=Region, OrganizationType4=Branch, Phonenumber=8089889910, PwdOriginal=test@123, Region=Indore, Run=Yes, Testrole=NewrolePlaywright, TestScenario=User_UserManagment, UNOriginal=IBU0001195, UserManagementPageRole=Admin, UsernameInUNPage=IBU0001192, ZoneCo=Ahmedabad}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
AddNewUserHeadOffice cannot be resolved to a variable
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
AddNewUserHeadOffice cannot be resolved to a variable

at com.test.user.UserManagementModule.UserManagement_TC.Add_new_user_page_Invalid_Email_Format(UserManagement_TC.java:590)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_new_user_page_Invalid_Email_Format -->
        <test-method is-config="true" signature="takeScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:57:56 IST" name="takeScreenshotOnFailure" finished-at="2024-12-28T20:57:56 IST" duration-ms="401" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Add_new_user_page_Invalid_Email_Format status=FAILURE method=UserManagement_TC.Add_new_user_page_Invalid_Email_Format(java.util.Map)[pri:19, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotOnFailure -->
        <test-method is-config="true" signature="afterEachTest()[pri:0, instance:com.test.user.UserManagementModule.UserManagement_TC@4116aac9]" started-at="2024-12-28T20:57:56 IST" name="afterEachTest" finished-at="2024-12-28T20:57:58 IST" duration-ms="1781" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterEachTest -->
      </class> <!-- com.test.user.UserManagementModule.UserManagement_TC -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
